packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -142,12 +194,73 @@ When these properties are used with conditional logic, you can trigger branding
+
+4. In the **Email address to send from**, enter the email address that you want to send the system notification emails from. This is what displays in the emails sent to your users.
+
+5. Enter the **Name of sender**. This name appears as the sender in the emails sent to your users.
+
+6. In the **Mail domain to send from** box, enter a unique mail domain that your organization has dedicated for Okta to send mail from. Later in this procedure, you add the unique mail domain to the SPF record to your DNS zone (the root domain) as an include-statement to show that you allow Okta to send mail from this unique mail domain.
+
+7. Click the **Save & View Required DNS Records** button to save your changes and view your org's DNS records that you need to update before your settings can take effect.
+
+8. Update your DNS records using the provided values.
+
+9. When you have updated your DNS records through your domain provider, click **I've updated the DNS records**. Okta begins polling your DNS records until it detects your updates (may take up to 24 hours). Your configuration is pending until the DNS updates are detected.
+
+    Alternatively, you can click **I will update the DNS records later**. Your records aren't polled and your configuration is incomplete until you update the relevant DNS records and click **I've updated the DNS records**. You can view the list of records that require an update at any time.
+
+10. Add the SPF record to your DNS zone (the root domain). An SPF record specifies the mail servers that your organization has authorized to send mail from your domain. If your root domain already has an SPF record, the following update can prevent spoofers from sending mail that mimics your domain.
+
+    For example, if you only send mail from Microsoft Office 365, your SPF record has an include-statement like this:
+
+    ```plain
+    example.com TXT    v=spf1 include:spf.protection.outlook.com -all
+    ```
+
+    To finish configuring your custom email domain, you must add another include-statement that specifies the mail domain that you specified in the **Mail domain to send from** box in step 6. This is also the host that appears in the first CNAME row in the DNS Records table in the Configure Email Sender dialog box.
+
+    ![CNAME Example](/img/CNAMEExample.png "CNAME table with an arrow pointing at the first CNAME row in the table")
+
+    Add the host to the existing record to configure a combined SPF record similar to this:
+
+    ```plain
+    example.com TXT    v=spf1 include:oktamail.example.com include:spf.protection.outlook.com -all
+    ```
+
+### Known Issues
+
+- You can't configure Okta to send emails through a domain that uses SendGrid. Instead, configure a subdomain on your DNS provider for custom Okta emails.
+
+- You can't have more than 10 DNS lookups in your SPF record.
+
+## See also
+
+Take a look at how to import users into Okta and manage their user profiles.
```

Comment:
I'd just delete this line altogether, or comment it out if we don't yet have a guide to link to. At the moment it just comes off as weird and unhelpful.